README.md

```diff
@@ -0,0 +1,198 @@
+<a name="readme-top"></a>
```

Comment:
# Backend review

- [ ] There are 2 model tests that are failing on my end:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/6577d325-45ec-41cf-bec5-759c01681450)

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/56eeb742-1d30-4ef0-b0fa-13f90b11a688)

Kindly fix those tests so your app's integrity is secured. 👍 

- [ ] The backend server project needs  to run on port 3001, according to your frontend code.   Kindly correct the command to run the server to:

```
rails server -p 3001
```

That way, your frontend will be able to consume the API. 👍